dozer-api/src/grpc/client_server.rs

```diff
@@ -134,7 +139,13 @@ impl ApiServer {
         let health_service = auth_middleware.layer(health_service);
 
         let mut auth_service = None;
-        if self.security.is_some() {
+        let dozer_master_secret = std::env::var("DOZER_MASTER_SECRET").ok();
+        let security = if self.security.is_none() && dozer_master_secret.is_some() {
+            Some(ApiSecurity::Jwt(dozer_master_secret.unwrap()))
+        } else {
+            self.security.clone()
+        };
```

Comment:
Looks like duplicated code with the lines above

Reply:
Yes. I meant the code is duplicated. We can create a function to avoid repeating ourselves.